akamai/resource_akamai_property_activation.go

```diff
@@ -208,7 +208,11 @@ func resourcePropertyActivationRead(d *schema.ResourceData, meta interface{}) er
 func resourcePropertyActivationUpdate(d *schema.ResourceData, meta interface{}) error {
 	log.Printf("[DEBUG] UPDATING")
 
-	property, e := getProperty(d)
+        log.Println("[DEBUG] Fetching property")
+        propertyID := d.Get("property").(string)
+        property := papi.NewProperty(papi.NewProperties())
+        property.PropertyID = propertyID
```

Comment:
Any reason not to just assign the property ID here, rather than to propertyID first?

Reply:
This was just a copy/paste from getProperty method. No reason not to merge lines 212 & 214.